test/e2e/.babelrc

```diff
@@ -0,0 +1,9 @@
+{
+	"presets": [
+		["@babel/preset-env", {
+			"targets": {
+				"node": "10.6.0"
```

Comment:
We should use `node: 'current'` here. That means that Node version that's used to run Babel will be used as transpilation target. Doesn't need to be changed on every Node upgrade.

But as I mentioned above, maybe we don't need Babel at all in e2e?

Reply:
We are really using babel for the import/export stuff. I am not very familiar with how it all works, but I think that is all we really need. I went ahead and set the rootMode and was able to remove the .babelrc file altogether. Thanks!